src/pages/Product-Listing/ProductListing.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Navbar } from "../../components";
+import { useStateContext } from "../../context/StateProvider";
+// import { Filters } from "./Filters";
+import "./product-listing.css";
+import { FaHeart } from "react-icons/fa";
+import { useNavigate } from "react-router-dom";
+
+export const ProductListing = () => {
+  const { productsList, productsDispatch } = useStateContext();
+  let navigate = useNavigate();
+
+  return (
+    <>
+      <Navbar />
+      <h3 className="text-center">Shop Our Collection </h3>
+      <div className="product-listing">
+        {/* <Filters /> */}
+        <div className="products">
+          {productsList.map((product) => {
+            const {
+              id,
+              name,
+              price,
+              marker,
+              image,
+              addedToWishlist,
+              addedToCart,
+            } = product;
```

Comment:
You can also destructure image here For e.g ```const {
              id,
              name,
              price,
              marker,
              image:{src:imageSrc,alt:imageAlt}, 
              addedToWishlist,
              addedToCart,
            } = product;```  or by simply doing ```const {
              id,
              name,
              price,
              marker,
              image:{src,alt}, 
              addedToWishlist,
              addedToCart,
            } = product;```

Reply:
Yup, i will do that. Thank you.